test/java/integration/BUILD

```diff
@@ -6,8 +6,7 @@ envoy_mobile_android_test(
         "AndroidEnvoyEngineStartUpTest.java",
     ],
     native_deps = [
-        "//library/common/jni:ndk_envoy_lib.so",
-        "//library/common/jni:ndk_envoy_lib.jnilib",
```

Comment:
I think we'll need to keep this for mac os x :(

Reply:
Oops - resurrected. Thanks.